client-gateway/bot/passenger.py

```
@@ -370,67 +221,243 @@ def handle_order_status(call):
+                f"🕐 _Оновлено: {updated_at}_"
+            )
+            
+            try:
+                if status not in ('COMPLETED', 'CANCELLED', 'ACTIVE'):
+                    markup = InlineKeyboardMarkup([
+                        [InlineKeyboardButton("🔄 Оновити статус", callback_data=f"refresh_status_{trip_id}")]
+                    ])
+                    await query.edit_message_text(response_text, parse_mode='Markdown', reply_markup=markup)
+                else:
+                    await query.edit_message_text(response_text, parse_mode='Markdown')
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)